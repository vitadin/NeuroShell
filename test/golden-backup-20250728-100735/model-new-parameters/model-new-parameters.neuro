%% Test model creation with parameters
\set[_echo_command="true"]
\model-new[provider="openai", base_model="gpt-3.5-turbo", temperature="0.8", max_tokens="2000", description="Advanced model for creative tasks"] creative-claude
\get[#model_id]
\get[#model_name]
\get[#model_provider]
\get[#model_base]
\get[#model_param_count]
\get[#model_description]
\get[_output]