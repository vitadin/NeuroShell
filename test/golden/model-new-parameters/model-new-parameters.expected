Setting _echo_command = true
%%> "\\model-new[provider=\"openai\", base_model=\"gpt-3.5-turbo\", temperature=\"0.8\", max_tokens=\"2000\", description=\"Advanced model for creative tasks\"] creative-claude"
Created model 'creative-claude' (ID: 00000001, Provider: openai, Base: gpt-3.5-turbo)
%%> "\\llm-client-get[provider=openai]"
LLM client ready: openai:b10394de (configured: true)
%%> "\\get[#model_id]"
#model_id = 00000001-0000-4000-8000-000000000001
%%> "\\get[#model_name]"
#model_name = creative-claude
%%> "\\get[#model_provider]"
#model_provider = openai
%%> "\\get[#model_base]"
#model_base = gpt-3.5-turbo
%%> "\\get[#model_param_count]"
#model_param_count = 2
%%> "\\get[#model_description]"
#model_description = Advanced model for creative tasks
%%> "\\get[_output]"
_output = LLM client ready: openai:b10394de